applications/StructuralMechanicsApplication/custom_response_functions/response_utilities/adjoint_local_stress_response_function.cpp

```diff
@@ -37,10 +39,23 @@ namespace Kratos
             mIdOfLocation = ResponseSettings["stress_location"].GetInt();
             KRATOS_ERROR_IF(mIdOfLocation < 1) << "Chose a 'stress_location' > 0. Specified 'stress_location': " << mIdOfLocation << std::endl;
         }
+
+        if(ResponseSettings.Has("consider_particular_solution"))
+            mAddParticularSolution = ResponseSettings["consider_particular_solution"].GetBool();
```

Comment:
Just for convenience: can you use similar names for the json setting and the variable in cpp?